src/main/scala/com/gu/support/workers/model/state/FailureHandlerState.scala

```diff
@@ -0,0 +1,13 @@
+package com.gu.support.workers.model.state
+
+import java.util.UUID
+
+import com.gu.support.workers.model._
+
+case class FailureHandlerState(
```

Comment:
It's an interesting point, I think before these were in the monthly contributions project so the namespace didn't matter so much. Maybe we should move them now before we put anything live?